homeassistant/components/google_generative_ai_conversation/config_flow.py

```diff
@@ -156,58 +164,90 @@ async def async_step_reauth_confirm(
             },
         )
 
-    @staticmethod
-    def async_get_options_flow(
-        config_entry: ConfigEntry,
-    ) -> OptionsFlow:
-        """Create the options flow."""
-        return GoogleGenerativeAIOptionsFlow(config_entry)
+    @classmethod
+    @callback
+    def async_get_supported_subentry_types(
+        cls, config_entry: ConfigEntry
+    ) -> dict[str, type[ConfigSubentryFlow]]:
+        """Return subentries supported by this integration."""
+        return {"conversation": ConversationSubentryFlowHandler}
```

Comment:
Are we sure that conversation is the thing we're adding here? Conversation sounds like a HA specific term and not necessarily a user facing one

Reply:
It's more like, will we ever see more entities or features added to a single "instance"? Because if it's just a model, why don't we call it model.

Reply:
The model is not even exposed to the user by default, because it is too complicated and not something a user should deal with:

<img width="601" alt="image" src="https://github.com/user-attachments/assets/960927a4-2310-4ad7-9522-b45b55f16907" />

Reply:
Oh and no, 1 subentry = 1 entity. We wouldn't want multiple entities under 1 subentry.